pkg/util/clusterdata/clusterdata.go

```diff
@@ -58,16 +58,38 @@ type bestEffortEnricher struct {
 }
 
 func (e *bestEffortEnricher) Enrich(ctx context.Context, ocs ...*api.OpenShiftCluster) {
```

Comment:
the changes here allow enrich to receive an arbitrary number of `ocs` to enrich.  Up to 10 are enriched concurrently each with a 10s timeout.